Sources/StreamChatUI/CommonViews/BaseViews.swift

```diff
@@ -49,11 +50,22 @@ public extension Customizable where Self: UIViewController {
     }
 }
 
+extension UIConfigProvider where Self: _View {
+    public func uiConfigDidRegister() {
+        if isInitialized {
+            log
+                .assertationFailure(
+                    "`UIConfig` was assigned after the view has been already initialized. This is most likely caused by assigning the custom `UIConfig` instance after the view has been added to the view hierarchy, or after the view's subviews have been initialized already. This is undefined behavior and should be avoided."
+                )
```

Comment:
nit: We tend to format things like this by multiple concatenated strings to keep the readability up:
```swift
log.assertationFailure(
    "`UIConfig` was assigned after the view has been already initialized. This is most likely caused by assigning " +
    "the custom `UIConfig` instance after the view has been added to the view hierarchy, or after the view's subviews " +
    "have been initialized already. This is undefined behavior and should be avoided."
)
```

Reply:
message ✅  let's merge the #935 first 👍